Makefile

```diff
@@ -2,12 +2,12 @@
 
 lint:
 	@luacheck lua/lualine
-	@luacheck tests
+	@luacheck lua/tests
 
 format:
 	@for file in `find -name '*.lua'`;do lua-format $$file -i; done;
 
 test:
-	@nvim --headless -u tests/minimal_init.lua -c "PlenaryBustedDirectory tests/ { minimal_init = './tests/minimal_init.lua' }"
```

Comment:
I'm actuallu against putting tests in lua. As it unnesserily adds them to user's `path` . You should already be able to do `require'lualine'` As it's in path . If you want to add things from tests to path you can apply the patch for minimal_init.lua 7fd434f76f2c2d06c03d75ea1ebe8b55eb8eec83 .